activity_heartbeating/my_workflow.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'temporalio/workflow'
+require_relative 'my_activities'
+
+module ActivityHeartbeating
+  class MyWorkflow < Temporalio::Workflow::Definition
+    def execute
+      # Execute the activity with a 5-minute timeout and 3-second heartbeat timeout
+      Temporalio::Workflow.execute_activity(
+        MyActivities::FakeProgress,
+        1000, # 1 second sleep interval
+        start_to_close_timeout: 5 * 60,
+        heartbeat_timeout: 3,
+        # Wait for activity cancellation completion
+        cancellation_type: Temporalio::Workflow::ActivityCancellationType::WAIT_CANCELLATION_COMPLETED
+      )
+    rescue Temporalio::Error::CanceledError
```

Comment:
Does this `rescue` work? I'd expect it'd be a `Temporalio::Error::ActivityError` with a cause of `Temporalio::Error::CanceledError`

Reply:
You're correct, I missed that the line wasn't being logged, as the effect is the same anyway.